libs/tiled-bitmap/tools/measure-framerate-callbacks.cc

```diff
@@ -39,14 +39,22 @@ static void on_hide(GtkWidget*, gpointer) { gtk_main_quit(); }
 
 static gboolean on_expose(GtkWidget* widget, GdkEventExpose*, gpointer)
 {
-  cairo_t* cr = gdk_cairo_create(gtk_widget_get_window(widget));
+  cairo_region_t* re = cairo_region_create();
```

Comment:
We'll probably see this pattern a lot. Could we create a RAII style helper class for this?